demos/ci-cd-ops-role-sdp/DemoSetupOutlineandNotes.adoc

```diff
@@ -0,0 +1,282 @@
+
+$ oadm policy add-role-to-user view dev -n ops-custom-eap6-proj
+
+#This command gives the dev user edit access to the ticket monster image build project
+
+$ oadm policy add-role-to-user edit dev -n ticket-monster-proj
+----
+* This command is key. #It gives the ticket-monseter-proj access to pull the custom image from the eap project#
+[source,]
+----
+$ oc policy add-role-to-group system:image-puller system:serviceaccounts:ticket-monster-proj -n ops-custom-eap6-proj
+----
+
+==== Login and create the App
+
+* Log in as the ops user and create the application, ...let it complete  
+[source,bash]
+----
+$ oc login -u ops  
+$ oc new-app http://registry.access.redhat.com/jboss-eap-6/eap-openshift~https://github.com/themoosman/ops-custom-eap6.git --name=ops-custom-eap6
+
+----
+
+* Login as the dev user and create the app, using the image created by the eap project
+
+[source,bash]
+----
+$ oc login -u dev  
+$ oc new-app ops-custom-eap6-proj/ops-custom-eap6:latest~https://github.com/themoosman/ticket-monster-ose-s2i-build.git --name=ticket-monster && \
+oc expose service ticket-monster --name=ticket-monster-route --hostname=<Add Hostname Here>
+----
+=== Demo Setup Commands -The “group” method
+
+This method creates two users and two groups
+
+* On the master logged in as system:admin issue the following commands
+
+[source,]
+----
+
+#Give tsar cluster admin privileges, ops gets edit privileges
+
+$ oadm policy add-cluster-role-to-user cluster-admin tsar
+
+
+#Create new groups (ops-edit, dev-view, dev-edit)
+
+$ oadm groups new ops-edit
+
+$ oadm groups new dev-view
+
+$ oadm groups new dev-edit
+
+
+# Add users to the groups
+
+$ oadm groups add-users ops-edit ops && \
+
+oadm groups add-users dev-view dev && \
+
+oadm groups add-users dev-edit dev
+
+
+#Grant access to the different groups
+#This command gives the ops-edit group edit access to the eap project
+
+$ oadm policy add-role-to-group edit ops-edit -n ops-custom-eap6-proj
+
+
+#This command gives the dev-view group view access to the eap project
+
+$ oadm policy add-role-to-group view dev-view -n ops-custom-eap6-proj
+
+
+#This command gives the dev-edit group edit access to the ticket monster image build project
+
+$ oadm policy add-role-to-group edit dev-edit -n ticket-monster-proj
+
+
+#This command is key. 
+#It gives the ticket-monster-proj access to pull the custom image from the eap project
+
+$ oc policy add-role-to-group system:image-puller system:serviceaccounts:ticket-monster-proj -n ops-custom-eap6-proj
+
+
+#View the final roles
+
+$ oc describe policyBindings :default -n ops-custom-eap6-proj
+$ oc describe policyBindings :default -n ticket-monster-proj
+
+----
+
+==== Login and create the App
+* Log in as the ops user and create the application, ...let it complete  
+[source,bash]
+----
+$ oc login -u ops  
+$ oc new-app http://registry.access.redhat.com/jboss-eap-6/eap-openshift~https://github.com/themoosman/ops-custom-eap6.git --name=ops-custom-eap6
```

Comment:
Which project are you creating this new application in. Can you either explain this in the instructions, switch to the project, or pass in the -n flag to the command